src/main/java/ledger/user_interface/ui_controllers/TransactionTableView.java

```diff
@@ -60,6 +59,10 @@ public void handle(CellEditEvent<TransactionModel, String> t) {
             try {
                 TransactionModel model = t.getTableView().getItems().get(t.getTablePosition().getRow());
                 String amountToSetString = t.getNewValue();
+                if (InputSanitization.isInvalidAmount(amountToSetString)) {
+                    setupErrorPopup("Provided amount is invalid", new Exception());
```

Comment:
In this case, we arn't inside TransactionTableView anymore. Using this would give us an instance of EventHandler